cron/monthly/verify-plans.js

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env node
+import '../../server/env';
+
+/**
+ * Makes sure all collectives.plan are up-to-date, downgrading and cancelling plans if needed.
+ */
+
+// Only run on the first of the month
+const today = new Date();
+if (process.env.NODE_ENV === 'production' && today.getDate() !== 1) {
+  console.log('NODE_ENV is production and today is not the first of month, script aborted!');
+  process.exit();
+}
+
+process.env.PORT = 3066;
+
+import { Op } from 'sequelize';
+import { compact, findKey, get, groupBy, map, values, pipe } from 'lodash/fp';
```

Comment:
Why not just use `lodash` ?

Reply:
I used this for composing the functions to eval `EXISTING_PLANS_SLUGS`.
Returning this to normal `lodash` after replacing that with plain JS.